specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-01-14-preview/desktopvirtualization.json

```diff
@@ -3870,6 +3870,58 @@
         "updateErrorMessage": {
           "type": "string",
           "description": "The error message."
+        },
+        "sessionHostHealthCheckResults": {
```

Comment:
What is the typical size of this array? Do you think all items would fit in a single request?

Reply:
we have around 10 health checks on average 400 bytes each. Although the list can grow. We think the size is still acceptable for a request.

Reply:
If the list can grow, then does it makes sense to add pagination?

Reply:
i talked to the ack end team. They said it won't grow much. We can address if it comes. We control list size internally.